### PR TITLE
FOSSC-69-webrtchcks-adapter

### PR DESCRIPTION
**Pull request by dierberg**

### CODE REVIEW

**Comment by dks-kraus** [comment]
The reviewer in the readme.md is missing. Everything else is looking fine.

File: analysed-packages/webrtcHacks-adapter/version-7.3.0/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+## Creator
+
+Sascha Dierberg <dierberg@dresearch-fe.de>
+
+## Reviewers
+
+The information was reviewed by:
+
+* [Reviewer]
```
Comment (dks-kraus) [inline analysed-packages/webrtcHacks-adapter/version-7.3.0/README.md]: The reviewer in the readme.md is missing. Everything else is looking fine.

**Review by dks-kraus — CHANGES_REQUESTED** [review]

File: analysed-packages/webrtcHacks-adapter/version-7.3.0/README.md (hunk 1 above)
Comment (dierberg) [reply analysed-packages/webrtcHacks-adapter/version-7.3.0/README.md]: Mario Schuknecht added as reviewer

**Review by dks-kraus — APPROVED** [review]
👍

File: analysed-packages/webrtcHacks-adapter/version-7.3.0/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+https://github.com/webrtcHacks/adapter/archive/v7.3.0.tar.gz
+
+## Package URL (purl)
+
+pkg:github/webrtcHacks/adapter@v7.3.0
+
+## Creator
+
+Sascha Dierberg <dierberg@dresearch-fe.de>
```
Comment (ParianGol) [inline analysed-packages/webrtcHacks-adapter/version-7.3.0/README.md]:
```suggestion
Sascha Dierberg
```

File: analysed-packages/webrtcHacks-adapter/version-7.3.0/webrtcHacks-adapter-7.3.0-SPDX2TV.spdx [hunk 3]
```diff
@@ -0,0 +1,1161 @@
...
+SPDXID: SPDXRef-item461562
+FileChecksum: SHA1: 52fe98e039ff7126843823d511716dd51d6397d6
+FileChecksum: SHA256: d1bc4875c51e33387506210c6996a8e3e61c8c468cd07afbfb7eaeb6ac45f3f1
+FileChecksum: MD5: a5e912c4c1edd771ddc4ee6052b47ef9
+LicenseConcluded: LicenseRef-BSD-3-Clause
+LicenseInfoInFile: LicenseRef-BSD
+LicenseInfoInFile: LicenseRef-bsd-new
+FileCopyrightText: <text> Copyright (c) 2016 The WebRTC project authors.
+Copyright (c) 2016 The WebRTC project authors.
+Copyright (c) 2016 The WebRTC project authors.
```
Comment (ParianGol) [inline analysed-packages/webrtcHacks-adapter/version-7.3.0/webrtcHacks-adapter-7.3.0-SPDX2TV.spdx:91]:
```suggestion
```

Comment (ParianGol) [reply analysed-packages/webrtcHacks-adapter/version-7.3.0/webrtcHacks-adapter-7.3.0-SPDX2TV.spdx:91]: This copyright repeated several times (see till line 77)

File: analysed-packages/webrtcHacks-adapter/version-7.3.0/webrtcHacks-adapter-7.3.0-SPDX2TV.spdx [hunk 4]
```diff
@@ -0,0 +1,1161 @@
...
+FileChecksum: SHA256: 9484b474ca001597224b9e14ad5cd34bd8d7793837a085f657e6b3a512abd028
+FileChecksum: MD5: 6b37869e44bc7df44fa50662a0e9d028
+LicenseConcluded: LicenseRef-BSD-3-Clause
+LicenseInfoInFile: LicenseRef-BSD
+LicenseInfoInFile: LicenseRef-bsd-new
+FileCopyrightText: <text> Copyright (c) 2016 The WebRTC project authors.
+Copyright (c) 2016 The WebRTC project authors.
+Copyright (c) 2016 The WebRTC project authors.
+Copyright (c) 2018 The adapter.js project authors.
+Copyright (c) 2016 The WebRTC project authors.
```
Comment (ParianGol) [inline analysed-packages/webrtcHacks-adapter/version-7.3.0/webrtcHacks-adapter-7.3.0-SPDX2TV.spdx:217]: repeated copyrights

File: analysed-packages/webrtcHacks-adapter/version-7.3.0/webrtcHacks-adapter-7.3.0-SPDX2TV.spdx [hunk 5]
```diff
@@ -0,0 +1,1161 @@
...
+
+FileName: webrtcHacks-adapter-v7.3.0.tar.gz/webrtcHacks-adapter-v7.3.0.tar/adapter-7.3.0/release/adapter_no_edge_no_global.js
+SPDXID: SPDXRef-item461557
+FileChecksum: SHA1: 12a4a0d3b4a31468cc17fc859f0f9980313222c2
+FileChecksum: SHA256: ade8ed8d593af1b7e600762ac0204a59b447e1201e1d381e395836af34e8fb46
+FileChecksum: MD5: 0e78b06df1ccd45fdb3f05cd21bc8aa3
+LicenseConcluded: LicenseRef-BSD-3-Clause
+LicenseInfoInFile: LicenseRef-BSD
+LicenseInfoInFile: LicenseRef-bsd-new
+FileCopyrightText: <text> Copyright (c) 2016 The WebRTC project authors.
```
Comment (ParianGol) [inline analysed-packages/webrtcHacks-adapter/version-7.3.0/webrtcHacks-adapter-7.3.0-SPDX2TV.spdx]: repeated copyrights ...

File: analysed-packages/webrtcHacks-adapter/version-7.3.0/webrtcHacks-adapter-7.3.0-SPDX2TV.spdx [hunk 6]
```diff
@@ -0,0 +1,1161 @@
...
+FileName: webrtcHacks-adapter-v7.3.0.tar.gz/webrtcHacks-adapter-v7.3.0.tar/adapter-7.3.0/release/adapter.js.gz/adapter.js
+SPDXID: SPDXRef-item461561
+FileChecksum: SHA1: b69c544a130a0f8c7753407259a8937e15c71b4c
+FileChecksum: SHA256: 551a6790404fe05891f81364df19129e3e2ea60b20db79f4c8104b273c7d78cc
+FileChecksum: MD5: 161a366b2bae740bf1f74df958f8c402
+LicenseConcluded: LicenseRef-BSD-3-Clause
+LicenseInfoInFile: LicenseRef-BSD
+LicenseInfoInFile: LicenseRef-bsd-new
+FileCopyrightText: <text> Copyright (c) 2016 The WebRTC project authors.
+Copyright (c) 2016 The WebRTC project authors.
```
Comment (ParianGol) [inline analysed-packages/webrtcHacks-adapter/version-7.3.0/webrtcHacks-adapter-7.3.0-SPDX2TV.spdx]: repeated copyrights

File: analysed-packages/webrtcHacks-adapter/version-7.3.0/webrtcHacks-adapter-7.3.0-SPDX2TV.spdx [hunk 7]
```diff
@@ -0,0 +1,1161 @@
...
+FileName: webrtcHacks-adapter-v7.3.0.tar.gz/webrtcHacks-adapter-v7.3.0.tar/adapter-7.3.0/release/adapter_no_global.js
+SPDXID: SPDXRef-item461556
+FileChecksum: SHA1: 77cc2c34a5fd6fdd29046f0bc9f875ab64e3f352
+FileChecksum: SHA256: 3df4244abe827792683eccefbab29fcd0a0b222a1ec5244a311db2a6ecb74cc5
+FileChecksum: MD5: f6bddee2fdcebc5e8e92ea1a7bc36b46
+LicenseConcluded: LicenseRef-BSD-3-Clause
+LicenseInfoInFile: LicenseRef-BSD
+LicenseInfoInFile: LicenseRef-bsd-new
+FileCopyrightText: <text> Copyright (c) 2016 The WebRTC project authors.
+Copyright (c) 2016 The WebRTC project authors.
```
Comment (ParianGol) [inline analysed-packages/webrtcHacks-adapter/version-7.3.0/webrtcHacks-adapter-7.3.0-SPDX2TV.spdx]: repeated copyrights

**Review by ParianGol — COMMENTED** [review]
It looks good to me apart from minor repeated copyrights.

**Pull request merged**